client/extensions/woocommerce/woocommerce-services/state/shipping-label/selectors.js

```diff
@@ -41,6 +41,13 @@ import {
 import { isEnabled } from 'config';
 import { ACCEPTED_USPS_ORIGIN_COUNTRY_CODES } from './constants';
 import getAddressValues from 'woocommerce/woocommerce-services/lib/utils/get-address-values';
+import {
+	areLocationsLoaded,
+	_getSelectorDependants,
```

Comment:
The new selectors are in this file because they are specific for the `shipping labels` feature. The `getOriginCountryNames()`, for example, depends on the value of the `international labels` feature flag to filter the country list. I figured that putting these new selectors alongside the general `locations` ones would introduce more confusion.